qemux86-64.robot

```diff
@@ -39,17 +39,21 @@ Checking if device comes online in 60s (Trying every 10s)
   Wait Until Keyword Succeeds    6x    10s    Device "${device_uuid}" is online
 Verify if resin-info@tty1.service is active
   Check if service "resin-info@tty1.service" is running using socket "unix\#/tmp/console.sock"
-Check that backup files are not found in the image
-  Check that backup files are not found in the "${image}"
 Git pushing to application
-  Push "${application_repo}":"${application_commit}" to application "${application_name}"
+  Push "${application_repo}":"${application_commit}" to application "${application_name}" from "/tmp/${application_name}"
```

Comment:
Lets remove this key word all together and use the containing parts 
i.e two lines 1) Git clone 2) git push